charts/aws-efs-csi-driver/templates/node-daemonset.yaml

```diff
@@ -47,13 +50,13 @@ spec:
       {{- with .Values.node.affinity }}
       affinity: {{- toYaml . | nindent 8 }}
       {{- end }}
-      hostNetwork: true
+      hostNetwork: {{ .Values.node.hostNetwork}}
```

Comment:
This value will need a default in the values file and it would be a good idea to use [default](https://helm.sh/docs/chart_template_guide/function_list/#default) here.